app/views/runs/_title.slim

```diff
@@ -1,50 +1,67 @@
+      a.dropdown-item.text-secondary href="https://one.livesplit.org/#/splits-io/#{run.id36}" target='_blank' LiveSplit One
+      - (Run.exportable_programs - [timer]).each do |timer|
+        a.dropdown-item.text-secondary href=download_path(run, timer.to_sym) = timer
+    - if run.category
+      - if !run.belongs_to?(current_user) && current_user.try(:runs?, run.category)
+        li
+          a.btn.btn-outline-light href=compare_path(current_user.pb_for(run.category), run)
+            i.fas.fa-chart-bar>
+            span Compare to mine
+      - if run.belongs_to?(current_user) && run != (run.category.best_known_run(timing) || run)
+        li
+          a.btn.btn-outline-success  href=compare_path(run, run.category.best_known_run(timing))
+            i.fas.fa-chart-bar>
+            span Compare to Best on Site
+  li
+    a.btn.btn-outline-success href=run_stats_path(run, params: {timing: params[:timing]})
+      i.fas.fa-chart-pie>
+      span Stats
+  - if run.srdc_id.present?
     li
-      a.fake-link#download-link onclick='toggle_download_menu()' ↓ download
-      - if run.category
-        - if !run.belongs_to?(current_user) && current_user.try(:runs?, run.category)
-          li = link_to 'compare to my pb', compare_path(current_user.pb_for(run.category), run)
-        - if run.best_known?(timing)
-          li ✓ best on record
-        - elsif run.pb?
-          - if run.belongs_to?(current_user) && run.category.best_known_run(timing).present?
-            li = link_to 'compare to best on record', compare_path(run, run.category.best_known_run(timing))
-          li ✓ pb
-    - if run.srdc_id.present?
-      li = link_to 'speedrun.com', run.srdc_url
-    - if run.video_url.present?
-      li = link_to 'proof', run.video_url
-    li = link_to 'stats', run_stats_path(run, params: {timing: params[:timing]})
-    - if can?(:edit, run)
-      li = link_to 'edit', edit_run_path(run)
-    - if run.user.nil?
-      li#claim-nav-link-container style='display: none;'
-        - if current_user.present?
-          = link_to "claim", '#', id: 'claim-nav-link', data: {confirm: "Claim this run as #{current_user.twitch_display_name}?"}
+      a.btn.btn-outline-light target='_blank' href=run.srdc_url
+        i.fas.fa-trophy>
+        span Speedrun.com
+  - if run.video_url.present?
+    - uri = URI.parse(run.video_url)
+    li
+      a.btn.btn-outline-light target='_blank' href=run.video_url
+        - if uri.host.match?(/^(www\.)?(twitch\.tv)$/)
+          i.fab.fa-twitch>
+        - elsif uri.host.match?(/^(www\.)?(youtube\.com|youtu\.be)$/)
+          i.fab.fa-youtube>
         - else
-          = link_to "sign in to claim", '/auth/twitch'
+          i.fas.fa-video>
+        span Video
+  - if can?(:edit, run)
+    li
+      a.btn.btn-outline-success href=edit_run_path(run)
+        i.fas.fa-edit>
+        span Edit
+  - if run.user.nil?
+    li#claim-nav-link-container style='display: none;'
```

Comment:
The claim link is unstyled, but seems a bit redundant.  Could remove it if you wanted, but it shouldn't be left as is.

![2018-06-09_23-39-31](https://user-images.githubusercontent.com/7030913/41198934-724e898a-6c3e-11e8-9df8-93a34b0e8b33.png)

Reply:
I should note that the giant claim box is right below the claim link, it's probably in another one of the screenshots.

Reply:
I put both in on purpose when I made this feature: the giant box is an attention-getter, but if you don't want to claim you can dismiss the box with the X. If you do, the link in nav serves as a way to still claim it later.

But I agree even though each has their purpose that it feels redundant. I won't try to handle this now though since it's current behavior in production, but I'll fix the styling on the link to be a button like the others.

Reply:
Ah, I didn't realize that dismissing the claim box would make it stay gone.  It makes perfect sense to keep it then.  Just needs to be styled on.